src/crimson/osd/osd_operations/snaptrim_event.cc

```diff
@@ -428,6 +428,7 @@ SnapTrimObjSubEvent::process_and_submit(ObjectContextRef head_obc,
 
   auto [submitted, all_completed] = co_await pg->submit_transaction(
 	  std::move(clone_obc),
+	  nullptr,
```

Comment:
This is confusing and might be error prune. submit_transaction expects obc and clone_obc, we pass the clone_obc as obc while clone_obc is nullptr. Renaming as suggested earlier might help